runtime/compiler/ilgen/J9ByteCodeIterator.cpp

```diff
@@ -611,8 +607,8 @@ const uint8_t TR_J9ByteCodeIterator::_byteCodeFlags[] =
                              0x01, // J9BCmonitorexit
                              0x00, // J9BCwide
                              0x01, // J9BCasyncCheck --- TODO: Is this the right size?
-                             0x03, // J9BCaconst_init
-                             0x03, // J9BCwithfield
+                             0x03, // BCunknown
+                             0x03, // BCunknown
```

Comment:
I think these two entries should be removed. . .  but let me double-check that.

Reply:
I tried removing these originally but what getting an error.

Reply:
I see these two lines were added in https://github.com/eclipse-openj9/openj9/pull/7991/files#diff-16ccd09398e392e4a6b8168ba45d9b05115abad7220fc067e37ad5df2ce9aa92 without other modifications related to these two opcode. Interesting, they couldn't be removed. Is it a build error or a test error?